src/rtctools/data/util.py

```diff
@@ -0,0 +1,42 @@
+"""Module with utilities for processing data."""
```

Comment:
We typically don't have this first-line comments in RTC-Tools modules

Reply:
I'll remove it.